src/cocoa/toga_cocoa/libs/__init__.py

```diff
@@ -2,5 +2,4 @@
 from .core_graphics import *
 from .core_text import *
 from .foundation import *
-from .quartz import *
```

Comment:
Is this an orphan from previous? Or am I missing where this was used?

Reply:
Ah. I see now, deleted file below. (still not sure why it was deleted, perhaps that will become clear)

Reply:
The Quartz module was a copy-pasta artefact from the code I used to bootstrap Rubicon. Turns out we weren't using any of it.